### ISSUE TITLE
what parameter is shared between scenes?

### ISSUE THREAD

**Issue by yshen47** [autonomousvision/monosdf#59]
I am a bit confused about the pre-trained model you provide. Based on my understanding, every scene needs to be optimized with the loss defined independently. Then,, I am not sure why the pre-trained model can be adapted to all scenes in the DTU dataset. What parameters are generalized between different scenes? Thanks for your clarification!

**Comment by niujinshuchong** [comment]
Hi, we provided different pretrained models for different scenes. There are no shared parameters.
